Cargo.toml

```diff
@@ -100,7 +100,7 @@ serde = { version = "^1.0.190", optional = true }
 ntapi = { version = "0.4", optional = true }
 # Support a range of versions in order to avoid duplication of this crate. Make sure to test all
 # versions when bumping to a new release, and only increase the minimum when absolutely necessary.
-windows = { version = ">=0.54, <=0.57", optional = true }
+windows = { version = ">=0.54, <=0.57", optional = true, features = ["Win32_Foundation", "Win32_System_Diagnostics_ToolHelp"] }
```

Comment:
`"Win32_Foundation"` is already listed in `system` and `"Win32_System_Diagnostics_ToolHelp"` should be moved there as well.